data/fh/sections.json

```diff
@@ -417,10 +444,78 @@
     "name": "Realm of Endless Frost",
     "parent": "21",
     "edition": "fh",
+    "marker": "1",
     "objectives": [
       {
         "name": "Glowing Orb",
-        "health": "(L+2)xC"
+        "health": "(L+2)xC",
+        "actions": [
+          {
+            "type": "custom",
+            "value": "Whenever it suffers damage, all characters and monsters suffer %game.action.damage% 2"
```

Comment:
Please change to `%game.action.damage% 2` to  `%game.damage:2%` then it will be shown correctly. This is also important for correct view when using old GH style.